README.md

```diff
@@ -0,0 +1,31 @@
+# 2i2c Team Compass
+
+This repository contains the source files for the [2i2c Team Compass](https://team-compass.2i2c.org).
+For more information, see [our Team Compass landing page](https://team-compass.2i2c.org).
+
+## Build the team compass locally
+
+The Team Compass is built with [the Sphinx documentation engine](https://sphinx-doc.org).
+The easiest way to build the documentation in this repository is to use [the `nox` automation tool](https://nox.thea.codes/), a tool for quickly building environments and running commands within them.
+This ensures that your environment has all the dependencies needed to build the documentation.
+
+To do so, follow these steps:
+
+1. Install `nox`
+
+   ```console
+   $ pip install nox
+   ```
+2. Build the documentation:
+
+   ```console
+   $ nox -s docs_build
+   ```
+
+This should create a local environment in a `.nox` folder, build the documentation (as specified in the `noxfile.py` configuration), and the output will be in `_build/html`.
+
+To build live documentation that updates when you update local files, run the following command:
+
+```console
+$ nox -s docs_live
+```
```

Comment:
@choldgraf, maybe we could have something like this for the pilot-hubs repo as well?

Reply:
I would love that - the only challenge I ran into was that building the docs for pilot-hubs required installing an anaconda environment, and `nox` doesn't yet support native anaconda use. So it would be possible, but I think it'd mean that building the environment would be much slower because you'd have to manually install each time you built. I think there are some workarounds there though, I just haven't had time to try it out

Reply:
> nox doesn't yet support native anaconda use. 

I see `nox` in conda-forge: https://anaconda.org/conda-forge/nox, maybe I am missing something else?

Reply:
> maybe I am missing something else?

I guess the building time we are adding to the CI... do you know how much time that is?

Reply:
yep - they do support conda commands directly, but not yet building a nox environment from a conda environment. The main dev does seem open to it though! https://github.com/theacodes/nox/issues/260